Makefile

```diff
@@ -14,7 +14,11 @@ NODE_PREFIX=$(shell pwd)
 
 app_name=richdocuments
 project_dir=$(CURDIR)/../$(app_name)
-build_dir=$(CURDIR)/build/artifacts
+doc_files=README.md
```

Comment:
In the root folder there is also ``admin.php`` and ``settings.php``
What are those there for?
Should they be listed somewhere here so that they end up in the tarball?

Reply:
These are the old ways how settings panels used to be registered.

I checked info.xml and it seems it wasn't ported to the new way, so these files are required I'd say... seems this wasn't discovered yet